arch/arm/soc/atmel_sam/Kconfig

```diff
@@ -2,7 +2,22 @@
 #
 # Copyright (c) 2016 Piotr Mienkowski
 # SPDX-License-Identifier: Apache-2.0
-#
+
+config SOC_FAMILY_SAM
+	bool
+	# omit prompt to signify a "hidden" option
+	default n
+
+if SOC_FAMILY_SAM
+
+config SOC_FAMILY
+	string
+	default atmel_sam
+
+config WATCHDOG
```

Comment:
I believe this option should stay in Kconfig.defconfig, the rest looks good.

Reply:
ok, updated